CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ These changes were made in the following pull requests:
 
 - [#4376: Implement the Tudor crown favicons (v4.x)](https://github.com/alphagov/govuk-frontend/pull/4376)
 - [#4278: Implement the Tudor crown in the Header component (v4.x)](https://github.com/alphagov/govuk-frontend/pull/4278)
+- [#4677: Adjust the spacing of the new Tudor crown (v4.x)](https://github.com/alphagov/govuk-frontend/pull/4677)
```

Comment:
```suggestion
- [#4677: Adjust the spacing of the new Tudor crown (v4.x)](https://github.com/alphagov/govuk-frontend/pull/4677) - thanks to [Martin Jones](https://github.com/MartinJJones) for contributing this change
```

Credit where it's due 😉

Reply:
Awesome, thanks very much! 😃 

Hope it's OK, I also added a mention for Mónica Crusellas, the interaction Designer on our team who helped me with this change - https://github.com/alphagov/govuk-frontend/pull/4677/commits/f607a831661319a24d9de85700ce29851e695230

Reply:
Good with me!